http://dev.mutt.org/trac/ticket/3352
http://dev.mutt.org/trac/changeset/a6fddecdc5f5

Make mutt_yesorno use mutt_message to print query. Closes #3352.


Index: curs_lib.c
===================================================================
--- curs_lib.c	(revision 6035:31881f38ca1e)
+++ curs_lib.c	(revision 6043:a6fddecdc5f5)
@@ -224,5 +224,5 @@
   snprintf (answer_string, COLS + 1, " ([%s]/%s): ", def == M_YES ? yes : no, def == M_YES ? no : yes);
   answer_string_len = strlen (answer_string);
-  printw ("%.*s%s", COLS - answer_string_len, msg, answer_string);
+  mutt_message ("%.*s%s", COLS - answer_string_len, msg, answer_string);
   FREE (&answer_string);
 
